### PR TITLE
Add io.gitlab.JakobDev.jdSimpleAutostart

### PR DESCRIPTION
**Pull request by JakobDev**
### Please confirm your submission meets all the criteria

- [x] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [x] My pull request follows the instructions at [App Submission][submission].
- [x] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [x] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [x] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link:**
- [x] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge).
- [x] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)*

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

filesystem=xdg-config/autostart:rwt:
This is needed to edit the autostart entries

filesystem=host:ro:
I only need access to /etc/xdg/autostart, but neither filesystem=/etc/xdg/autostart:ro or filesystem=/etc:ro works, so I'm orced ro give full host access

### CODE REVIEW

**Comment by JakobDev** [comment]
bot, build io.gitlab.JakobDev.jdSimpleAutostart

**Comment by JakobDev** [comment]
bot, build io.gitlab.JakobDev.jdSimpleAutostart

File: io.gitlab.JakobDev.jdSimpleAutostart.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+base-version: '6.3'
+command: jdsimpleautostart
+cleanup-commands:
+  - /app/cleanup-BaseApp.sh
+build-options:
+  env:
+    - BASEAPP_REMOVE_PYWEBENGINE=1
+finish-args:
+  - --filesystem=xdg-config/autostart:rw
+  - --filesystem=host:ro
```
Comment (barthalion) [inline io.gitlab.JakobDev.jdSimpleAutostart.yaml:15]:
```suggestion
  - --filesystem=host-etc:ro

```

Comment (JakobDev) [reply io.gitlab.JakobDev.jdSimpleAutostart.yaml:15]: Didn't know that. Thanks.

**Comment by JakobDev** [comment]
bot, build io.gitlab.JakobDev.jdSimpleAutostart

**Comment by JakobDev** [comment]
@barthalion I now use `host-etc`. I also use create instead of rw for xdg-config/autostart now, so you have to whitelist this.

**Comment by barthalion** [comment]
Please submit it to https://github.com/flathub/flatpak-builder-lint/blob/master/flatpak_builder_lint/staticfiles/exceptions.json and I will merge it.

**Comment by barthalion** [comment]
bot, build io.gitlab.JakobDev.jdSimpleAutostart

**Pull request closed without merging**